.github/workflows/consume-release.yml

```diff
@@ -6,13 +6,44 @@ on:
   workflow_run:
     workflows: [Release]
     types: [completed]
+    branches: [develop]
+  workflow_dispatch:
+    inputs:
+      workflowRunId:
+        description: The workflow_run.id of the release workflow that has published artifacts
+        required: true
 
 jobs:
-  on-success:
+  get-images:
+    runs-on: ubuntu-latest
     environment: InfraK8s
+    steps:
+      - id: get-artifacts
+        name: 'Get artifacts from release workflow'
+        uses: actions/github-script@d50f485531ba88479582bc2da03ff424389af5c1 # v6.1.1
+        env:
+          AWS_PRIVATE_ECR_PATH: ${{ secrets.AWS_PRIVATE_ECR_PATH }}
+          WORKFLOW_RUN_ID: ${{ inputs.workflowRunId }}
```

Comment:
This needs to be conditional based on the trigger. I would test the following block, i havent tried it yet.

```
${{ inputs.workflowRunId  || github.event.workflow_run.id  }}

```